MXParallaxHeader/MXParallaxHeader.m

```diff
@@ -214,7 +214,13 @@ - (void)setBottomModeConstraints {
 
 - (void)layoutContentView {
     CGFloat minimumHeight = MIN(self.minimumHeight, self.height);
-    CGFloat relativeYOffset = self.scrollView.contentOffset.y + self.scrollView.contentInset.top - self.height;
+    CGFloat topInset;
+    if (@available(iOS 11.0, *)) {
+        topInset = self.scrollView.adjustedContentInset.top;
```

Comment:
if you add 
            topInset = self.scrollView.contentInsetAdjustmentBehavior = UIScrollViewContentInsetAdjustmentNever;
before that, it will work with translucent navigationBar

Reply:
Sorry for the late reply.
Having the header start directly below the nav bar was the intended behavior, as it is how the library behaves on iOS 10. The reason is that if you make the header start behind the nav bar, the top portion of the header (picture for example) will never be visible.